pulsar/transaction_test.go

```diff
@@ -458,7 +458,6 @@ func TestAckChunkMessage(t *testing.T) {
 	// Create transaction and register the send operation.
 	txn, err := client.NewTransaction(time.Hour)
 	require.Nil(t, err)
-	txn.(*transaction).registerSendOrAckOp()
```

Comment:
We do register in `internalSendAsync`. This call seems like a workaround we ever added as a HACK 🤣